solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -490,7 +490,7 @@ protected static boolean isSSLMode() {
   public static void useFactory(String factory) throws Exception {
     // allow calling more than once so a subclass can override a base class
     if (!changedFactory) {
-      savedFactory = System.getProperty("solr.DirectoryFactory");
+      savedFactory = System.getProperty("solr.directoryFactory");
```

Comment:
?

Reply:
Was the first thing I tried in the investigation. It's unrelated, but a typo nonetheless